packages/sonarwhal/src/lib/cli/rules/new-rule.ts

```diff
@@ -29,30 +28,31 @@ const copyCommonFiles = async (destination: string) => {
 };
 
 const generateRuleFiles = async (destination: string, data) => {
-    const commonFiles = [{
-        destination: path.join(destination, '.sonarwhalrc'),
-        path: path.join(__dirname, TEMPLATE_PATH, 'rule-config.hbs')
-    },
-    {
-        destination: path.join(destination, 'README.md'),
-        path: path.join(__dirname, TEMPLATE_PATH, 'rule-doc.hbs')
-    },
-    {
-        destination: path.join(destination, 'package.json'),
-        path: path.join(__dirname, TEMPLATE_PATH, 'rule-package.hbs')
-    },
-    {
-        destination: path.join(destination, 'src', `index.ts`),
-        path: path.join(__dirname, TEMPLATE_PATH, 'rule-index.hbs')
-    }];
+    const commonFiles = [
+        {
+            destination: path.join(destination, '.sonarwhalrc'),
+            path: path.join(__dirname, TEMPLATE_PATH, '.sonarwhalrc.hbs')
+        },
+        {
+            destination: path.join(destination, 'src', `index.ts`),
+            path: path.join(__dirname, TEMPLATE_PATH, 'index.ts.hbs')
+        },
+        {
+            destination: path.join(destination, 'package.json'),
+            path: path.join(__dirname, TEMPLATE_PATH, 'package.json.hbs')
+        },
+        {
+            destination: path.join(destination, 'README.md'),
+            path: path.join(__dirname, TEMPLATE_PATH, 'readme.md.hbs')
+        }];
 
     const ruleFile = {
-        destination: path.join(destination, 'src', 'rules'),
-        path: path.join(__dirname, TEMPLATE_COMMON_PATH, 'rule-script.hbs')
+        destination: path.join(destination, 'src'),
+        path: path.join(__dirname, TEMPLATE_PATH, 'rule.ts.hbs')
     };
     const testFile = {
         destination: path.join(destination, 'tests'),
-        path: path.join(__dirname, TEMPLATE_COMMON_PATH, 'rule-test.hbs')
+        path: path.join(__dirname, TEMPLATE_PATH, 'tests.ts.hbs')
```

Comment:
`tests.ts.hbs` needs to be updated (e.g.: `{{#if external}}` isn't needed anymore).

Reply:
Done